module/icp/io/skein_mod.c

```diff
@@ -231,6 +231,19 @@ skein_mod_init(void)
 
 int
 skein_mod_fini(void) {
+	int ret;
+
+	if (skein_prov_handle != 0) {
+		if ((ret = crypto_unregister_provider(skein_prov_handle)) !=
+		    CRYPTO_SUCCESS) {
```

Comment:
I styled the code like this in order to match the existing code of Illumos's KCF (and therefore the rest of the ICP). For instance, [here ](https://github.com/illumos/illumos-gate/blob/master/usr/src/uts/common/crypto/io/sha2_mod.c#L226-L241) is the `_init()` code from Illumos's sha2_mod.c. Would you still like me to change it?